app/services/evidence/rules/20240429122128_restraint_or_freezing_order.rb

```diff
@@ -0,0 +1,20 @@
+module Evidence
+  module Rules
+    class RestraintOrFreezingOrder < Rule
+      include Evidence::RuleDsl
+
+      key :restraint_freezing_order_31
+      group :none
+
+      client do |crime_application|
+        (crime_application.capital&.has_frozen_income_or_assets == 'yes' ||
+          crime_application.income&.has_frozen_income_or_assets == 'yes') || false
```

Comment:
What's the || false for?

Reply:
@hiboabd oh I see. They can both return nil. Forgive me.

Reply:
@mseedat-moj it does error if `nil` is returned but i don't think my condition will ever return `nil`. Happy to keep as a precaution however